### PR TITLE
Modifications to CT classification and gear refactoring

### PR DESCRIPTION
**Pull request by bhalchandranoopur**
- Modifications to CT classification 
   Reconstruction window check

- Gear refactoring

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -2,11 +2,11 @@
   "name": "dicom-metadata-classification",
   "label": "Metadata Classification: DICOM",
   "description": "This gear classifies DICOM images and updates the classification metadata of the input file",
-  "version": "1.0.1",
+  "version": "1.0.1_sbx_dev8",
```
Comment (kalebdfischer) [inline manifest.json]: Should be fine to just bump the minor version and use 1.1.0

Comment (bhalchandranoopur) [reply manifest.json]: Added change to address this

File: common_utils.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import logging
+import flywheel
+import re
+import string
+from fnmatch import fnmatch
+import pydicom
+import zipfile
+import pandas as pd
+import pprint
+import ast
```
Comment (NPann) [inline common_utils.py]: only `re` and `ast` seems to be used in this module

File: run.py [hunk 3]
```diff
@@ -13,334 +12,32 @@
 import zipfile
 import pandas as pd
 import pprint
+import ast
 
-import classify_CT
+import dicom_processor
+import common_utils
```
Comment (NPann) [inline run.py]: import can be cleanup
```
import os
import json
import sys
import logging
import flywheel
import pprint
import dicom_processor
import CT_classifier
import MR_classifier
```

**Comment by NPann** [comment]
Only minor comments on cleaning up the imports for improve lisibility/(potentially) overhead

File: CT_classifier.py [hunk 4]
```diff
@@ -1,139 +1,171 @@
-import classification_from_label
 import re
 import ast
```
[the remark this answers is not in the document]
Comment (bhalchandranoopur) [reply CT_classifier.py]: Added change to address this

File: Dockerfile [hunk 5]
```diff
@@ -12,8 +12,14 @@ RUN pip install -r requirements.txt
...
-COPY classification_from_label.py .
-RUN chmod +x classification_from_label.py
+COPY dicom_processor.py .
+RUN chmod +x dicom_processor.py
+COPY common_utils.py .
+RUN chmod +x common_utils.py
+COPY MR_classifier.py .
+RUN chmod +x MR_classifier.py
+COPY CT_classifier.py .
+RUN chmod +x CT_classifier.py
```
[the remark this answers is not in the document]
Comment (bhalchandranoopur) [reply Dockerfile]: Added change to address this

File: common_utils.py (hunk 2 above)
Comment (bhalchandranoopur) [reply common_utils.py]: Added change to address this

File: run.py (hunk 3 above)
Comment (bhalchandranoopur) [reply run.py]: Added change to address this

**Pull request merged**